.github/workflows/terraform-integration-tests.yml

```diff
@@ -49,6 +49,23 @@ jobs:
           echo "Hazelcast GCP jar is: " target/hazelcast-gcp-*-SNAPSHOT.jar
           cp target/hazelcast-gcp-*-SNAPSHOT.jar ~/lib/hazelcast-gcp.jar
 
+      # FIND LATEST MANAGEMENT CENTER VERSION
```

Comment:
Isn't there any simpler way to find the latest MC version? Could you check with the Management Center team? CC: @emre-aydin

Reply:
We have already discussed this on Slack. The best place would be the GitHub repository and the tags in there for the main Management Center project but unfortunately it's a private project. Maybe you could configure your job to have a secret and access the private repo though I'm not sure if it's an option for you.

Reply:
@hasancelik @mtyazici Isn't it possible to use the secret token which has access to read MC repo?